### DISCUSSION TITLE
Server hosting using HTTPS

### DISCUSSION THREAD

**Discussion by woola168** [elsa-workflows/elsa-core#2818] [Q&A] [answered]
I want to host a dashboard with server,can elsa use https for hosting ?
Now I can enter the website using http but not https

**Comment by sfmskywalker** [comment, marked as answer]
Absolutely. In fact, that's what I do by default. Any ASP.NET application can be hosted using HTTPS. Elsa is no different, since you would typically use ASP.NET to host Elsa.
